### DISCUSSION TITLE
the hover didn't work in ui.card?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1330] [Q&A]
```py
ui.card().classes('bg-white hover:bg-sky-500')
```

**Comment by falkoschindler** [comment]
The problem is:
"bg-white" is a valid Quasar class and Quasar sets background colors with the "!important" suffix.
"bg-sky-500, in contrast, is a Tailwind class and yields a CSS color setting without "!important".
Therefore the white background overrules the sky color.

You can either make the hover color "!important":
```py
ui.card().classes('bg-white hover:!bg-sky-500')
```
or avoid using a Quasar class for defining the initial background:
```py
ui.card().classes('bg-[#fff] hover:bg-sky-500')
```
